recipes/ua-nodeset/all/conandata.yml

```
@@ -0,0 +1,4 @@
+sources:
+  "1.02":
```

The version needs to be updated in 2 places (`config.yml` for the global index, and in `conandata.yml` for the sources).
```suggestion
  "1.02-2021-07-21":
```